### PR TITLE
Update documentation Microsoft SQL Server App according to changes in Helm Chart v3

### PR DESCRIPTION
**Pull request by kkujawa-sumo**
## Purpose of this pull request

Update documentation Microsoft SQL Server App according to changes in Helm Chart v3

Issue number:  https://jira.kumoroku.com/jira/browse/SUMO-206934

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/integrations/microsoft-azure/sql-server.md [hunk 1]
```diff
@@ -70,9 +70,15 @@ Sumo Logic supports collection of logs and metrics data from SQL Server in both
 
 <TabItem value="k8s">
 
-In Kubernetes environments, we use the Telegraf Operator, which is packaged with the Kubernetes collection ([learn more](/docs/send-data/collect-from-other-data-sources/collect-metrics-telegraf/telegraf-collection-architecture)). The diagram below illustrates how data is collected from SQL Server in Kubernetes environments. In the architecture shown below, there are four services that make up the metric collection pipeline: Telegraf, Prometheus, Fluentd and FluentBit.<br/><img src={useBaseUrl('img/integrations/microsoft-azure/sqlk8s.png')} alt="sqlk8s.png" />
+In Kubernetes environments, we use the Telegraf Operator, which is packaged with the Kubernetes collection ([learn more](/docs/send-data/collect-from-other-data-sources/collect-metrics-telegraf/telegraf-collection-architecture)). The diagram below illustrates how data is collected from SQL Server in Kubernetes environments. In the architecture shown below, there are four services that make up the metric collection pipeline: Telegraf, Telegraf Operator, Prometheus, and [Sumo Logic Distribution for OpenTelemetry Collector](https://github.com/SumoLogic/sumologic-otel-collector).
```
Comment (JainM6) [inline docs/integrations/microsoft-azure/sql-server.md:73]:
```suggestion
In Kubernetes environments, we use the Telegraf Operator, which is packaged with the Kubernetes collection ([learn more](/docs/send-data/collect-from-other-data-sources/collect-metrics-telegraf/telegraf-collection-architecture)). The diagram below illustrates how data is collected from SQL Server in Kubernetes environments. In the architecture shown below, there are four services that make up the metric collection pipeline: Telegraf, Telegraf Operator, Prometheus, and [Sumo Logic Distribution for OpenTelemetry Collector](/docs/send-data/sumo-distribution-opentelemetry/).
```

File: docs/integrations/microsoft-azure/sql-server.md [hunk 2]
```diff
@@ -70,9 +70,15 @@ Sumo Logic supports collection of logs and metrics data from SQL Server in both
...
 
-The first service in the pipeline is Telegraf. Telegraf collects metrics from SQL Server. Note that we’re running Telegraf in each pod we want to collect metrics from as a sidecar deployment: i.e. Telegraf runs in the same pod as the containers it monitors. Telegraf uses the [SQL Server input plugin](https://github.com/influxdata/telegraf/tree/master/plugins/inputs/sqlserver) to obtain metrics. (For simplicity, the diagram doesn’t show the input plugins.) The injection of the Telegraf sidecar container is done by the Telegraf Operator. We also have Fluentbit that collects logs written to standard out and forwards them to FluentD, which in turn sends all the logs and metrics data to a Sumo Logic HTTP Source.
+<br/><img src={useBaseUrl('img/integrations/microsoft-azure/sqlk8s.png')} alt="sqlk8s.png" />
```
Comment (JainM6) [inline docs/integrations/microsoft-azure/sql-server.md:75]:
```suggestion
```

File: docs/integrations/microsoft-azure/sql-server.md (hunk 1 above)
Comment (JainM6) [inline docs/integrations/microsoft-azure/sql-server.md:73]:
```suggestion
In Kubernetes environments, we use the Telegraf Operator, which is packaged with the Kubernetes collection ([learn more](/docs/send-data/collect-from-other-data-sources/collect-metrics-telegraf/telegraf-collection-architecture)). The diagram below illustrates how data is collected from SQL Server in Kubernetes environments. In the architecture shown below, there are four services that make up the metric collection pipeline: Telegraf, Telegraf Operator, Prometheus, and [Sumo Logic Distribution for OpenTelemetry Collector](https://github.com/SumoLogic/sumologic-otel-collector). <br/><img src={useBaseUrl('img/integrations/microsoft-azure/sqlk8s.png')} alt="sqlk8s.png" />
```

File: docs/integrations/microsoft-azure/sql-server.md [hunk 3]
```diff
@@ -70,9 +70,15 @@ Sumo Logic supports collection of logs and metrics data from SQL Server in both
...
+
+The first service in the pipeline is Telegraf. Telegraf collects metrics from SQL Server. Note that we’re running Telegraf in each pod we want to collect metrics from as a sidecar deployment: i.e. Telegraf runs in the same pod as the containers it monitors. Telegraf uses the [SQL Server input plugin](https://github.com/influxdata/telegraf/tree/master/plugins/inputs/sqlserver) to obtain metrics. For simplicity, the diagram doesn’t show the input plugins.
```
Comment (JainM6) [inline docs/integrations/microsoft-azure/sql-server.md:77]:
```suggestion
The first service in the pipeline is Telegraf. Telegraf collects metrics from SQL Server. Note that we are running Telegraf as a sidecar deployment in each pod from which we want to collect metrics. This means that Telegraf is running in the same pod as the containers it monitors. Telegraf uses the [SQL Server input plugin](https://github.com/influxdata/telegraf/tree/master/plugins/inputs/sqlserver) to obtain metrics. For simplicity, the diagram doesn’t show the input plugins.
```

**Review by JainM6 — CHANGES_REQUESTED** [review]
Thank you @kkujawa-sumo. Minor formatting change.

File: docs/integrations/microsoft-azure/sql-server.md (hunk 1 above)
Comment (kkujawa-sumo) [reply docs/integrations/microsoft-azure/sql-server.md]: changed

File: docs/integrations/microsoft-azure/sql-server.md (hunk 2 above)
Comment (kkujawa-sumo) [reply docs/integrations/microsoft-azure/sql-server.md]: changed

File: docs/integrations/microsoft-azure/sql-server.md (hunk 3 above)
Comment (kkujawa-sumo) [reply docs/integrations/microsoft-azure/sql-server.md]: changed

**Comment by kkujawa-sumo** [comment]
@JainM6 please take another look

File: docs/integrations/microsoft-azure/sql-server.md [hunk 4]
```diff
@@ -70,9 +70,13 @@ Sumo Logic supports collection of logs and metrics data from SQL Server in both
...
+In Kubernetes environments, we use the Telegraf Operator, which is packaged with the Kubernetes collection ([learn more](/docs/send-data/collect-from-other-data-sources/collect-metrics-telegraf/telegraf-collection-architecture)). The diagram below illustrates how data is collected from SQL Server in Kubernetes environments. In the architecture shown below, there are four services that make up the metric collection pipeline: Telegraf, Telegraf Operator, Prometheus, and [Sumo Logic Distribution for OpenTelemetry Collector](https://github.com/SumoLogic/sumologic-otel-collector). <br/><img src={useBaseUrl('img/integrations/microsoft-azure/sqlk8s.png')} alt="sqlk8s.png" />
```
[the remark this answers is not in the document]
Comment (kkujawa-sumo) [reply docs/integrations/microsoft-azure/sql-server.md:73]: Yes, it links the repository,  I think that at this moment we don't have better source with information about Sumo Logic Distribution for OpenTelemetry Collector

[the remark this answers is not in the document]
Comment (JainM6) [reply docs/integrations/microsoft-azure/sql-server.md:73]: https://help.sumologic.com/docs/send-data/sumo-distribution-opentelemetry/. We have a doc for this.

Comment (JainM6) [inline docs/integrations/microsoft-azure/sql-server.md:73]:
```suggestion
In Kubernetes environments, we use the Telegraf Operator, which is packaged with the Kubernetes collection ([learn more](/docs/send-data/collect-from-other-data-sources/collect-metrics-telegraf/telegraf-collection-architecture)). The diagram below illustrates how data is collected from SQL Server in Kubernetes environments. In the architecture shown below, there are four services that make up the metric collection pipeline: Telegraf, Telegraf Operator, Prometheus, and [Sumo Logic Distribution for OpenTelemetry Collector](docs/send-data/sumo-distribution-opentelemetry/). <br/><img src={useBaseUrl('img/integrations/microsoft-azure/sqlk8s.png')} alt="sqlk8s.png" />
```

[the remark this answers is not in the document]
Comment (JainM6) [reply docs/integrations/microsoft-azure/sql-server.md]: I have add this doc path `docs/send-data/sumo-distribution-opentelemetry/`

**Pull request merged**